safe/gui/tools/peta_jakarta_dialog.py

```diff
@@ -537,3 +528,26 @@ def reject(self):
         # add our own logic here...
 
         super(PetaJakartaDialog, self).reject()
+
+    def download(self, url, output_path):
+        """Download file from url and write to output path.
+
+        :param url: URL of the zip bundle.
+        :type url: str
+
+        :param output_path: Path of output file,
+        :type output_path: str
+        """
+        downloader = FileDownloader(url, output_path)
+        try:
+            result = downloader.download()
+        except IOError as ex:
```

Comment:
ok, I see you copy/pasted from the OSM Downloader.
But doing this is a bad practice. We shouldn't re-raise an exception like this, because you are raising a new exception and we loose the previous traceback.

For now, let's remove this try/except. You can also do it in the OSM Downloader please.
Tim would like this PR to be merged.

But ideally, we should manage exception like last time. Let's do that later, ok ?